LoopCaregiver/LoopCaregiver/Views/Actions/OverrideView.swift

```diff
@@ -62,7 +62,7 @@ struct OverrideView: View {
                         
                         do {
                             //TODO: Set appropriate display symbol
```

Comment:
Great catch here. This is why I like to include the unit in the name -- like durationInSeconds -- although this override preset model comes from Nightscout directly which uses "duration".